Find a functionally equivalent code:android.widget.CalendarView.getDate:COMMENT
Method Modifier: public      
Comment:/**
 * Gets the selected date in milliseconds since January 1, 1970 00:00:00 in
 * {@link TimeZone#getDefault()} time zone.
 *
 * @return The selected date.
 */

@@ -1,3 +1,3 @@
 {
-    return mAdapter.mSelectedDate.getTimeInMillis();
+    return mDelegate.getDate();
 }

